src/main/java/com/powsybl/openloadflow/network/impl/LfShuntImpl.java

```diff
@@ -60,6 +75,6 @@ public Evaluable getQ() {
 
     @Override
     public void updateState() {
-        shuntCompensator.getTerminal().setQ(q.eval() * PerUnit.SB);
+        terminal.setQ(q.eval() * PerUnit.SB);
```

Comment:
@geofjamg I have some doubts about this line because in case of a static var compensator in control mode `VOLTAGE` the update of reactive power should be the one of the generator and not this one. But if the static var compensator becomes PQ, it could remain a shunt and the update should be this one.